docs/src/components/Header/Header.js

```diff
@@ -6,6 +6,12 @@ import { Link as MDSLink } from '@innovaccer/design-system';
 
 const Header = ({ relativePagePath }) => {
   const items = useHeaderItems();
+  const checkactive = (label) => {
+    let res = false;
+    const pagePath = relativePagePath.split('/');
+    if(pagePath[1]===label.toLowerCase()) res = true; 
+    return res; 
+  }
```

Comment:
This approach will not work for "mobile" pages. For e.g. http://localhost:8000/mobile/components/button/content/